include/mega/command.h

```diff
@@ -273,10 +273,11 @@ class MEGA_API CommandPutUA : public Command
 class MEGA_API CommandGetUA : public Command
 {
     string uid;
-    attr_t at;  // attribute type
+    attr_t at;      // attribute type
+    const char *ph; // public handle for preview mode
```

Comment:
Saving the `const char *` directly would require to take the ownership of the buffer received in the constructor or at least to know and manage the lifetime of that buffer. I think that it's much better to copy the value to a `std::string` like we do in other commands.